src/database/Database.cpp

```diff
@@ -211,6 +211,9 @@ Database::applySchemaUpgrade(unsigned long vers)
     case 15:
         mApp.getPersistentState().setRebuildForType(TRUSTLINE);
         break;
+    case 16:
```

Comment:
We haven't released schema version 15 yet, so you can do this in that version too.

Reply:
Good point. Done.